applications/GeoMechanicsApplication/python_scripts/element_test/triaxial.py

```diff
@@ -54,9 +57,14 @@ def run(self):
 
         return tensors, yy_strain, vol_strain, von_mises_values, mean_stress_values
 
-    def _load_stage_parameters(self, n_stages=2):
-        parameter_files = [os.path.join(self.work_dir, f'ProjectParameters_stage{i + 1}.json') for i in range(n_stages)]
-        return [Kratos.Parameters(open(f).read()) for f in parameter_files]
+    def _load_stage_parameters(self):
+        parameter_files = [os.path.join(self.work_dir, 'ProjectParameters.json')]
+        parameters = []
+        for f in parameter_files:
+            with open(f, 'r') as file:
+                parameters.append(Kratos.Parameters(file.read()))
+        return parameters
```

Comment:
Since it's now single stage, we can simplify if I'm correct:
```suggestion
    def _load_stage_parameters(self):
        file = os.path.join(self.work_dir, 'ProjectParameters.json')
        return [Kratos.Parameters(file.read()]
```